molgenis-dataexplorer/src/main/resources/js/dataexplorer.js

```diff
@@ -94,7 +94,7 @@ function($, molgenis, settingsXhr) {
 		var items = [];
 		items.push('<ul class="nav nav-tabs pull-left" style="width: 100%" role="tablist">');
 		$.each(modules, function() {
-			var href = molgenis.getContextUrl() + '/module/' + this.id+'?entity=' + entity;
+			var href = molgenis.getContextUrl() + '/module/' + this.id+'?entity=' + entity + "&" + window.location.search;
```

Comment:
You can beter use this code:
			var wls = window.location.search;
			var href = molgenis.getContextUrl() + '/module/' + this.id;
			if(wls.indexOf('?entity=') === -1) href += '?entity=' + entity;
			href += window.location.search;
Instead of:
var href = molgenis.getContextUrl() + '/module/' + this.id+'?entity=' + entity + "&" + window.location.search;

This code will create two times (?entity=' + entity) we it already exists.